setup.py

```diff
@@ -36,8 +36,8 @@
         "pydantic",
     ],
     extras_require={
-        "test": ["responses", "pytest", "pytest-describe", "sklearn"],
-        "ml": ["sklearn", "torch", "torchvision"],
+        "test": ["responses", "pytest", "pytest-describe", "scikit-learn"],
+        "ml": ["scikit-learn", "torch", "torchvision"],
```

Comment:
Is this necessary?

Reply:
The import that was there is wrong see - https://pypi.org/project/sklearn/

Reply:
Leads to failed tests if run at certain times of day.